opts/opts.go

```diff
@@ -215,6 +215,7 @@ func ValidatePath(val string) (string, error) {
 	return val, nil
 }
 
+// FIXME should it return error as it is always nil ?
```

Comment:
Well, I was asking if this was ok before doing it :smile_cat:. Now that I know it is I will fix it :wink:.